CODEOWNERS

```diff
@@ -807,6 +807,8 @@ scripts/build/gen_image_info.py           @tejlmand
 /tests/bluetooth/controller/              @cvinayak @thoh-ot @kruithofa @erbr-ot @sjanc @ppryga
 /tests/bluetooth/bsim_bt/                 @alwa-nordic @jhedberg @Vudentz @wopu-ot
 /tests/bluetooth/bsim_bt/bsim_test_audio/ @jhedberg @Vudentz @wopu-ot @Thalley @asbjornsabo
+/tests/bluetooth/bsim_bt/bsim_test_mesh/  @jhedberg @Vudentz @wopu-ot
+/tests/bluetooth/mesh_shell               @jhedberg @Vudentz @sjanc
```

Comment:
This is missing @PavelVPV from both entries.

Reply:
The mesh team in Nordic has planed to go through the codeowners and maintainers for mesh after this has been merged to find out who will get responsibilities for this from us.

Reply:
I just thought that at a bare minimum these new entries should at least include the Zephyr Mesh maintainer from the start.

Reply:
I agree, let's please add @PavelVPV initially so that we have someone there that actually works on mesh, and then you can override that later once you have figured it out internally

Reply:
I'm fine with this

Reply:
I think this should be done in a separate PR. The commit message explains the operation that was performed to arrive at the result.